digdag-docs/src/operators.rst

```diff
@@ -2031,3 +2031,176 @@ Parameters
 
   *Note:* Enabling retries might cause the target endpoint to receive multiple duplicate HTTP requests. Thus retries should only be enabled if duplicated requests are tolerable. E.g. when the outcome of the HTTP request is *idempotent*.
 
+
+
+emr>: Amazon Elastic Map Reduce
+-------------------------------
+
+The **emr>:** operator can be used to run EMR jobs, create clusters and submit steps to existing clusters.
+
+For detailed information about EMR, see the `Amazon Elastic MapReduce Documentation <https://aws.amazon.com/documentation/elastic-mapreduce/>`_.
+
+
+.. code-block:: yaml
+
+    +emr_job:
+      emr>:
+      cluster:
+        name: my-cluster
+        ec2:
+          key: my-ec2-key
+          master:
+            type: m3.2xlarge
+          core:
+            type: m3.xlarge
+            count: 10
+        logs: s3://my-bucket/logs/
+      staging: s3://my-bucket/staging/
+      steps:
+        - type: spark
+          application: pi.py
+        - type: spark-sql
+          query: queries/query.sql
+          result: s3://my-bucket/results/${session_uuid}/
+        - type: script
+          script: scripts/hello.sh
+          args: [hello, world]
+
+Secrets
+~~~~~~~
+
+:command:`aws.emr.access-key-id, aws.access-key-id`
```

Comment:
According to the implementation of EmrOperatorFactory.EmrOperator#credentials, `access_key_id` is correct? We don't convert `-` with `_` in secrets, right?